elliott/elliottlib/cli/verify_cve_trackers_cli.py

```
@@ -0,0 +1,417 @@
+            processed_files.append(file_path)
+            if group is None:
+                group, assembly = file_group, file_assembly
+            elif (file_group, file_assembly) != (group, assembly):
+                raise ValueError(
+                    f"Shipment MR contains mixed assemblies: expected {group}/{assembly}, "
+                    f"found {file_group}/{file_assembly} in {file_path}"
+                )
+        except ValueError:
+            raise
+        except Exception:
+            continue
```

Ruff (0.15.18)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)